config/routes.rb

```diff
@@ -2,5 +2,8 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  get "/users", to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- Nice Job 🥇 

![Capture](https://user-images.githubusercontent.com/63972670/197560903-0552d19c-4c42-48cf-acb0-a7f0cae3df9a.JPG)

Reply:
Thsnk you @OLIPLICHE  :pray: